app/src/main/res/values/strings.xml

```diff
@@ -1501,6 +1501,10 @@
     <string name="saved_logins_sort_strategy_last_used">Last used</string>
     <!-- Content description (not visible, for screen readers etc.): Sort saved logins dropdown menu chevron icon -->
     <string name="saved_logins_menu_dropdown_chevron_icon_content_description">Sort logins menu</string>
+    <!-- Content Description for a login's web address text field -->
```

Comment:
minor nit for the 2 comments here. Can we use the lower case "Description" to be consistent?
 
```suggestion
    <!-- Content description for a login's web address text field -->
```

Reply:
done